src/prefect/client/client.py

```diff
@@ -683,16 +643,22 @@ def get_cloud_url(self, subdirectory: str, id: str, as_user: bool = True) -> str
         # returns "https://cloud.prefect.io/my-tenant-slug/flow-run/424242-ca-94611-111-55"
         ```
         """
-        # Generate direct link to Cloud flow
-        tenant_slug = self.get_default_tenant_slug(as_user=as_user)
+        # Generate direct link to UI
+        if "prefect.io" in urlparse(prefect.config.cloud.api).netloc:
```

Comment:
Jw, why were all of the `prefect.io` checks added?

Reply:
For easier local development ;)